### PR TITLE
docs: added _Pushing an image to kind_

### PR DESCRIPTION
**Pull request by themr0c**
* docs: pushing an image to kind.
* fix: don't ignore `kind` at it has impact on what we can commit in the website.

references: https://github.com/containers/podman-desktop/issues/1955*

### CODE REVIEW

File: website/docs/kubernetes/kind/deploying-an-image-to-kind.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 10
+title: Deploying an image to Kind
+description: Deploying an Podman image to your Kind cluster
```
Comment (benoitf) [inline website/docs/kubernetes/kind/deploying-an-image-to-kind.md]: AFAIK it's not a Podman image it's an OCI image or just 'image'

File: website/docs/kubernetes/kind/deploying-an-image-to-kind.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+#### Procedure
+
+1. Open **Podman Desktop dashboard > Images**.
+1. **<icon icon="fa-solid fa-search" size="lg" /> Search images**: `<your_image>`.
+1. Click **<icon icon="fa-solid fa-ellipsis-v" size="lg" /> > <icon icon="fa-solid fa-ellipsis-v" size="lg" /> Push image to Kind cluster**.
+1. Choose your kind cluster in the list.
+
+#### Verification
+
+* TODO
```
Comment (benoitf) [inline website/docs/kubernetes/kind/deploying-an-image-to-kind.md]: should be removed of completed ?

File: website/docs/kubernetes/kind/deploying-an-image-to-kind.md (hunk 1 above)
Comment (benoitf) [reply website/docs/kubernetes/kind/deploying-an-image-to-kind.md]: if you use `an Podman image` I'm unsure it's `an` but `a Podman image`

Comment (themr0c) [reply website/docs/kubernetes/kind/deploying-an-image-to-kind.md]: Does it work with an image stored on a Docker instance?

File: website/docs/kubernetes/kind/deploying-an-image-to-kind.md (hunk 2 above)
Comment (themr0c) [reply website/docs/kubernetes/kind/deploying-an-image-to-kind.md]: Completed. Yesterday my tests were not successful

File: website/docs/kubernetes/kind/deploying-an-image-to-kind.md (hunk 1 above)
Comment (themr0c) [inline website/docs/kubernetes/kind/deploying-an-image-to-kind.md]:
```suggestion
description: Deploying an image to your Kind cluster
```

File: website/docs/kubernetes/kind/deploying-an-image-to-kind.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+sidebar_position: 10
+title: Deploying an image to Kind
+description: Deploying an image to your Kind cluster
+keywords: [podman desktop, podman, containers, images, migrating, kubernetes]
+tags: [migrating-to-kubernetes, images]
+---
+
+# Deploying an image to your local Kind-powered Kubernetes cluster 
+
+With Podman Desktop, you can deploy a Podman image to your local Kind-powered Kubernetes cluster.
```
Comment (lstocchi) [inline website/docs/kubernetes/kind/deploying-an-image-to-kind.md]: `Podman image` should be `image` as done above

File: website/docs/kubernetes/kind/deploying-an-image-to-kind.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+#### Prerequisites
+
+* Your are using the Podman container engine.
```
Comment (lstocchi) [inline website/docs/kubernetes/kind/deploying-an-image-to-kind.md]: `Your` -> `You`

**Review by lstocchi — DISMISSED** [review]

File: website/docs/kubernetes/kind/deploying-an-image-to-kind.md (hunk 3 above)
Comment (themr0c) [inline website/docs/kubernetes/kind/deploying-an-image-to-kind.md]:
```suggestion
With Podman Desktop, you can deploy an image to your local Kind-powered Kubernetes cluster.
```

File: website/docs/kubernetes/kind/deploying-an-image-to-kind.md (hunk 4 above)
Comment (themr0c) [inline website/docs/kubernetes/kind/deploying-an-image-to-kind.md]:
```suggestion
* You are using the Podman container engine.
```

File: website/docs/kubernetes/kind/pushing-an-image-to-kind.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+tags: [migrating-to-kubernetes, images]
+---
+
+# Pushing an image to your local Kind-powered Kubernetes cluster
+
+With Podman Desktop, you can push an image to your local Kind-powered Kubernetes cluster.
+
+#### Prerequisites
+
+- You are using the Podman container engine.
```
Comment (jeffmaury) [inline website/docs/kubernetes/kind/pushing-an-image-to-kind.md]: That is not limited to Podman engines

File: website/docs/kubernetes/kind/pushing-an-image-to-kind.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+1. **<icon icon="fa-solid fa-search" size="lg" /> Search images**: `<your_image>:<your_tag>`.
+1. Click **<icon icon="fa-solid fa-ellipsis-v" size="lg" /> > <icon icon="fa-solid fa-ellipsis-v" size="lg" /> Push image to Kind cluster**.
+1. If you created many Kind clusters, select your Kind cluster from the list.
+
+#### Verification
+
+Kind does not enable you to list loaded images.
+Therefore, create a Pod that uses the loaded image.
+
+1. Create a `verify_my_image.yaml` Kubernetes YAML file on your workstation:
```
Comment (jeffmaury) [inline website/docs/kubernetes/kind/pushing-an-image-to-kind.md]: We need to tell that the placeholders have to be replaced by user values

**Review by jeffmaury — DISMISSED** [review]

File: website/docs/kubernetes/kind/pushing-an-image-to-kind.md (hunk 5 above)
Comment (themr0c) [inline website/docs/kubernetes/kind/pushing-an-image-to-kind.md]:
```suggestion
```

File: website/docs/kubernetes/kind/pushing-an-image-to-kind.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
```
Comment (themr0c) [inline website/docs/kubernetes/kind/pushing-an-image-to-kind.md]:
```suggestion
1. Create a `verify_my_image.yaml` Kubernetes YAML file on your workstation.
   Replace the placeholders:
   
   * Pod `name` and container `name` value must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character.
   * Container `image` value is the image you pushed.
   
```

File: website/docs/kubernetes/kind/pushing-an-image-to-kind.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+   
+   * Pod `name` and container `name` value must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character.
+   * Container `image` value is the image you pushed.
+   
+
+   ```yaml
+   apiVersion: v1
+   kind: Pod
+   metadata:
+   name: <verify-my-image>
```
Comment (jeffmaury) [inline website/docs/kubernetes/kind/pushing-an-image-to-kind.md]: Seems indentation is missing here and in the following lines

**Review by jeffmaury — CHANGES_REQUESTED** [review]

File: website/docs/kubernetes/kind/pushing-an-image-to-kind.md (hunk 8 above)
Comment (themr0c) [inline website/docs/kubernetes/kind/pushing-an-image-to-kind.md]:
```suggestion
     name: <verify-my-image>
```

File: website/docs/kubernetes/kind/pushing-an-image-to-kind.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+   spec:
+   containers:
```
Comment (themr0c) [inline website/docs/kubernetes/kind/pushing-an-image-to-kind.md]:
```suggestion
     containers:
```

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**